synapse/models/geospace.py

```diff
@@ -21,6 +23,17 @@
     (10.0, 'cm'),
 )
 
+def getGeospaceContext():
+    '''
+    Get a context object for handling floating point operations.
+
+    Returns:
+        decimal.Context: A pre-configured context object.
+    '''
+    c = decimal.Context()
+    c.prec = 9
```

Comment:
Can't this just be a global?

Reply:
it definitely needs to be a geospace related global for doing decimal math; thats just polish in comparison to the math.